server/verify.go

```diff
@@ -323,30 +324,51 @@ func makePool(certs []*x509.Certificate) *x509.CertPool {
 func VerifyGceTechnology(attestation *pb.Attestation, tech pb.GCEConfidentialTechnology, opts *VerifyOpts) error {
 	switch tech {
 	case pb.GCEConfidentialTechnology_NONE: // Nothing to verify
+		if opts.TEEOpts != nil {
+			return fmt.Errorf("TEEopts is non-nil")
```

Comment:
You're not adding any formatting here, but I think you should. The tech is in an event that should be surfaced as the reason this is an error, e.g., fmt.Errorf("memory encryption technology %v does not support TEEOpts", tech)

Reply:
Done.